cli/command/command.go

```diff
@@ -112,15 +110,11 @@ func (cm *Command) get(c *cli.Context, entityType string, getEntity func(string)
 		return err
 	}
 
-	entities := []entity.Entity{}
 	for _, id := range ids {
-		entity, err := getEntity(id)
-		if err != nil {
+		if err := getEntity(id); err != nil {
 			return err
 		}
-
-		entities = append(entities, entity)
 	}
 
-	return cm.Printer.PrintEntities(entities)
+	return nil
```

Comment:
So we've stripped out the `cm.Printer.PrintEntities(entities)` line, but I'm not seeing a replacement. How does the obtained entity get displayed to the user?